src/transactions/LumenEventReconciler.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+// Copyright 2025 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "transactions/EventManager.h"
+#include "xdr/Stellar-ledger.h"
+#include "xdr/Stellar-transaction.h"
+#include <vector>
+
+namespace stellar
+{
+
+struct LedgerTxnDelta;
```

Comment:
nit: there shouldn't be a circular dependency with LedgerTxn.h, so why not include it?

Reply:
True. Updated